src/Rst/RstParser.php

```diff
@@ -45,6 +45,7 @@ class RstParser
     const DIRECTIVE_RST_CLASS = '.. rst-class::';
     const DIRECTIVE_LITERALINCLUDE = '.. literalinclude::';
     const DIRECTIVE_CONTENTS = '.. contents::';
+    const DIRECTIVE_CODEIMPORT = '.. codeimport::';
```

Comment:
Note this isn't standard - this is a custom extension https://github.com/phpbench/phpbench/blob/master/docs/_extension/codeimport.py#L106

Reply:
Thanks, maybe we can make this configurable in the future